TAF_EasyRest/TestFramework/PageComponents/Moderator/UnapprovedRestaurantsPageComponent .cs

```diff
@@ -0,0 +1,37 @@
+﻿namespace TestFramework.PageComponents.Moderator
+{
+    public class UnapprovedRestaurantsPageComponent
+    {
+        private IWebDriver driver;
```

Comment:
Let's use property (private IWebDriver driver { get; }) here instead of the field. You use property in your ArchivedRestaurantsPageComponent

Reply:
Done.